tools/run-tests.py

```diff
@@ -76,32 +68,25 @@ def skip_if(condition, desc):
 
 # Test options for jerry-tests
 JERRY_TESTS_OPTIONS = [
-    Options('jerry_tests-es.next-debug',
-            OPTIONS_COMMON + OPTIONS_PROFILE_ESNEXT + OPTIONS_DEBUG + OPTIONS_STACK_LIMIT + OPTIONS_GC_MARK_LIMIT
+    Options('jerry_tests-es.next',
```

Comment:
@ossy-szeged Hi, it's this build configuration.
On master, it's only test jerry_tests-es.next-debug, not the jerry_tests-es build configuration

Reply:
Ah, good to know that we haven't run esnext release jerry tests on the CI ever. Bad news.
In this case I prefer --skip-list=regression-test-issue-3785.js for OSX release jerry-test command
until #4530 is fixed properly.

Reply:
Oh, I was think it's leaved not running intentionally